translations/default/messages.xlf

```diff
@@ -2378,190 +2212,6 @@ According to the Rec.30;Art.7(1)(c)</target>
       </trans-unit>
     </body>
   </file>
-  <file original="modules/referralprogram/referralprogram.php" source-language="en-US" target-language="en" datatype="plaintext">
```

Comment:
I suppose this was never meant to be here?

Reply:
I'd say it shouldn't, like `followup` and `ps_reminder` which are neither in the composer file nor in the TranslationTool module's list so I don't understand how they got in the first extract 🤷‍♂️ @matthieu-rolland any idea?

However @eternoendless I was thinking that maybe there are some modules that are not native but which we still need to integrate in the core catalog because they don't have their own XLF catalog yet. It rings a bell but I don't remember which modules we had to handle this way, do you remember maybe?

Reply:
Modules like autoupgrade [integrate their own translations](https://github.com/PrestaShop/autoupgrade/tree/dev/translations). Since 1.7.8 [modules can also use XLF for translations](https://devdocs.prestashop.com/8/modules/creation/module-translation/new-system/#distributing-your-translations).

Followup and reminder have been abandoned, though.